genai-on-vertex-ai/vertex-ai-search/Ingesting_unstructured_docuemtns_with_metadata.ipynb

```diff
@@ -0,0 +1,1054 @@
+{
```

Comment:
<ol><li>typo: fucntions</li><li>Should we add a sentence or two about layout parse and ancestor information? or link to the documentation</li><li>create_datastore function is below this cell (says above)</li></ol>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/applied-ai-engineering-samples/pull/67/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Reply:
<p>Done</p>